content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -444,6 +444,39 @@ securityContext:
 To assign SELinux labels, the SELinux security module must be loaded on the host operating system.
 {{< /note >}}
 
+### Speeding up SELinux volume relabeling
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+By default, the contrainer runtime recursively assigns SELinux label to all
+files on all Pod volumes. To speed up this process, Kubernetes can change the
+SELinux label of a volume instantly by using a mount option
+`-o context=<label>`.
+
+To benefit from this speedup, all these conditions must be met:
+
+* Alpha feature gates `ReadWriteOncePod` and `SELinuxMountReadWriteOncePod` must
```

Comment:
Are these gates documented?

Reply:
Added `SELinuxMountReadWriteOncePod` to docs/reference/command-line-tools-reference/feature-gates.md.
`ReadWriteOncePod` is already there.

Reply:
Thank.